interfaces/kubernetes_manifest/v0/schema.py

```diff
@@ -0,0 +1,49 @@
+"""This file defines the schemas for the provider and requirer sides of the kubernetes_manifest interface.
+
+Examples:
+    RequirerSchema:
+        unit: <empty>
+        app:
+            kubernetes_manifests: [
+                                        {
+                                            "apiVersion": "v1",
+                                            "kind": "Secret",
+                                            "metadata": {
+                                                "name": "seldon-rclone-secret",
+                                                "labels": {"user.kubeflow.org/enabled": "true"},
+                                            },
+                                            "stringData": {"RCLONE_CONFIG_MYS3_TYPE": "test"},
+                                        },
+                                        {
+                                            "apiVersion": "v1",
+                                            "kind": "Secret",
+                                            "metadata": {
+                                                "name": "mlpipeline-minio-artifact",
+                                                "labels": {"user.kubeflow.org/enabled": "true"},
+                                            },
+                                            "stringData": {"AWS_ACCESS_KEY_ID": "access_key"},
+                                        },
+                                    ]
+"""
+
+from typing import List
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel
+
+
+class KubernetesManifest(BaseModel):
+    """
+    Representation of a Kubernetes Object sent to Kubernetes Manifests.
+
+    Args:
+        manifest_content: the content of the Kubernetes manifest file
+    """
+
+    manifest_content: str
+
+
+class RequirerSchema(DataBagSchema):
+    """The schema for the requirer side of kubernetes_manifest interface."""
+
+    kubernetes_manifests: List[KubernetesManifest]
```

Comment:
you're supposed to have an 'unit' and/or an 'app' in here:
https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/ingress/v2/schema.py#L58

it needs to read:

```
class MyAppDatabag(BaseModel):
    kubernetes_manifests: List[KubernetesManifest]

class RequirerSchema(DataBagSchema):
    app: MyAppDatabag
```

the requirer schema represents "the requirer's side", which includes app and unit databags with their own schemas.

it's easier to think about it if you map it to show-relation's output
![image](https://github.com/canonical/charm-relation-interfaces/assets/6230162/88333d18-ddd3-4889-a1fa-4763dcb33966)